versioned_sidebars/version-v2.18.x-sidebars.json

```diff
@@ -388,6 +388,7 @@
               ]
             },
             "user-guide/api-mediation/configuration-customizing-the-api-catalog-ui",
+            "user-guide/api-mediation/configuration-at-tls-configuration",
```

Comment:
This is already in line 255. The issue is in the article itself where the comment at the top breaks it